src/overlays/actors/ovl_Bg_Gnd_Darkmeiro/z_bg_gnd_darkmeiro.h

```diff
@@ -10,10 +10,26 @@ typedef void (*BgGndDarkmeiroActionFunc)(struct BgGndDarkmeiro*, GlobalContext*)
 
 typedef struct BgGndDarkmeiro {
     /* 0x0000 */ DynaPolyActor dyna;
-    /* 0x0164 */ char unk_164[0x8];
+    /* 0x0164 */ u16 actionState;    /* Bitflags for whether a block or timer is on.
+                                        0x8 for timer 2, 0x4 for timer 1, 0x2 for blocks.   */
+    /* 0x0166 */ s16 timer1;         /* Timer for switch flag N+1. Blocks use it to count 
+                                        down to vanishing when their switch flag is cleared.*/
+    /* 0x0168 */ s16 timer2;         /* Timer for switch flag N+2. Blocks use it to control
+                                        their transparency (does not work).                 */
     /* 0x016C */ BgGndDarkmeiroActionFunc actionFunc;
 } BgGndDarkmeiro; // size = 0x0170
 
+typedef enum {
+    /* 0 */ MODE_ZERO,   // This does nothing and has a display list which apparently draws nothing.
+    /* 1 */ CLEAR_BLOCK, /* Clear blocks appear when their switch flag is set and disappear 64 frames
+                            after their flag is cleared. Clear blocks with flag 0x3F are always on.   */
+    /* 2 */ BLOCK_TIMER  /* A block timer with switch flag N reacts to switch flags N+1 and N+2 being
```

Comment:
You want to prefix those with eg `DARKMEIRO_TYPE_` so they don't clutter the global symbol list